public/keymaps/r/rmkeebs_rm_fullsize_default.json

```
@@ -10,7 +10,7 @@
       "KC_TAB",  "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",    "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",    "KC_LBRC", "KC_RBRC", "KC_BSLS",               "KC_DEL",  "KC_END",  "KC_PGDN",    "KC_P7",   "KC_P8",   "KC_P9",   "KC_PEQL",
       "KC_CAPS", "KC_A",    "KC_S",    "KC_D",    "KC_F",    "KC_G",    "KC_H",    "KC_J",    "KC_K",    "KC_L",    "KC_SCLN", "KC_QUOT",            "KC_ENT",                                                    "KC_P4",   "KC_P5",   "KC_P6",   "KC_PPLS",
       "KC_LSFT",            "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",    "KC_N",    "KC_M",    "KC_COMM", "KC_DOT",  "KC_SLSH",            "KC_RSFT",                          "KC_UP",                 "KC_P1",   "KC_P2",   "KC_P3",   "KC_DEL",
-      "KC_LCTL", "KC_LGUI", "KC_LALT",                                  "KC_SPC",                                   "KC_RALT", "KC_RGUI", "KC_APP",  "KC_RCTL",               "KC_LEFT", "KC_DOWN", "KC_RGHT",    "KC_P0",   "KC_00",   "KC_PDOT", "KC_PENT"
+      "KC_LCTL", "KC_LGUI", "KC_LALT",                                  "KC_SPC",                                   "KC_RALT", "KC_RGUI", "KC_APP",  "KC_RCTL",               "KC_LEFT", "KC_DOWN", "KC_RGHT",    "KC_P0",   "KC_P0",   "KC_PDOT", "KC_PENT"
```

I meant for this to be a macro keycode here.

Would you please throw this in above Line 6:

```json
  "macros": [
    [
      "00"
    ]
  ],
```

and swap the broken keycode to:

```suggestion
      "KC_LCTL", "KC_LGUI", "KC_LALT",                                  "KC_SPC",                                   "KC_RALT", "KC_RGUI", "KC_APP",  "KC_RCTL",               "KC_LEFT", "KC_DOWN", "KC_RGHT",    "KC_P0",   "MC_0",    "KC_PDOT", "KC_PENT"
```

?